files/jcl/ZWENOSEC.jcl

```diff
@@ -75,7 +75,7 @@
 //         SET PRODUCT=RACF          * RACF, ACF2, or TSS
 //*                     12345678
 //         SET ADMINGRP=ZWEADMIN     * group for Zowe administrators
-//         SET STCGROUP=&ADMINGRP.   * group for Zowe started tasks
+//         SET STCGRP=&ADMINGRP.     * group for Zowe started tasks
```

Comment:
minor nitpick, it looks better if we keep the alignment of the = signs

Reply:
Done